static/src/javascripts/projects/common/modules/commercial/dfp/apply-creative-template.js

```diff
@@ -75,10 +75,8 @@ define([
         var creativeConfig = fetchCreativeConfig();
 
         if (creativeConfig) {
-            return Promise.all([
```

Comment:
Fine, but then we beg the question: why exactly are we waiting for this operation to complete, what's going to potentially run next, and what side effects do we want to settle before then?

In the general case, for an async operation to only finish half of its work before resolving its promise seems a bitty and booby-trapped interface to me.